### PR TITLE
feat: add Connector docs to LB4 docs

### LINKED ISSUE #2598: Add "Connectors Reference" docs pages to LB4 docs
## Description / Steps to reproduce / Feature proposal
When working on https://github.com/strongloop/loopback-next/issues/1766, I realized that the docs pages under [Connectors references](https://loopback.io/doc/en/lb3/Connectors-reference.html) are still very relevant, and also because we're still using the existing connectors.  

I'd like to propose to copy these docs pages to LB4 docs. 
@strongloop/loopback-maintainers, what do you think? 

## Acceptance Criteria
- [ ] Add the [Connectors Reference](https://loopback.io/doc/en/lb3/Connectors-reference.html) including the subtree to LB4 docs. 
  - [ ] memory connector
  - [ ] database connector
      - can we assume all will be working in LB4? 
  - [ ] Other connectors 
      - Are everything under this category works the same way in LB4?  There's gap in email connector #1980 
  - [ ] community connectors

References:
- Sample of how it can be done: https://github.com/strongloop/loopback.io/blob/gh-pages/pages/en/lb3/Cloudant-connector.md.

_See [Reporting Issues](http://loopback.io/doc/en/contrib/Reporting-issues.html) for more tips on writing good issues_

### PR DESCRIPTION
**Pull request by hacksparrow**
Add Connector docs to LB4 docs. 

Required to fix https://github.com/strongloop/loopback-next/issues/2598.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,1004 @@
+{
```
Comment (dhmlau) [inline package-lock.json]: should we remove the `package-lock.json`?

**Review by raymondfeng — APPROVED** [review]

**Review by b-admike — COMMENTED** [review]
Nice. Do we need to copy the files in `update-readmes.sh` again though since we are doing it in the `update-lb4-docs.js` script?

**Comment by raymondfeng** [comment]
@hacksparrow Please fix the CI issue - https://travis-ci.org/strongloop/loopback.io/builds/566037056?utm_source=github_status&utm_medium=notification

**Comment by hacksparrow** [comment]
@b-admike 
> Nice. Do we need to copy the files in update-readmes.sh again though since we are doing it in the update-lb4-docs.js script?

They are working on different set of files:
- `update-readmes.sh` -> README files specifically from external repos
- `update-lb4-docs.js` -> Doc files which already exists in the `lb3` dir

File: update-readmes.sh [hunk 2]
```diff
@@ -8,6 +8,7 @@
 #   from the given github repo. If that branch is NOT master, then the
 #   branch name will be appended to the local readme file name.
 (cat <<LIST_END
+strongloop loopback-connector-mysql master
```
Comment (jannyHou) [inline update-readmes.sh:11]: nitpick: `strongloop loopback-connector-couchdb2 master` is also supported

**Review by jannyHou — APPROVED** [review]
👍

**Review by nabdelgadir — APPROVED** [review]

File: update-readmes.sh (hunk 2 above)
Comment (hacksparrow) [reply update-readmes.sh:11]: Let's add it after this one lands.

**Pull request merged**